frameworks/Crystal/amber/shard.lock

```diff
@@ -48,31 +48,27 @@ shards:
     github: stefanwille/crystal-redis
     version: 1.9.0
 
-  sentry:
-    github: samueleaton/sentry
-    version: 0.1.1
-
   shell-table:
     github: jwaldrip/shell-table.cr
     version: 0.9.2
 
   slang:
     github: jeromegn/slang
-    version: 1.7.0
+    commit: 777e2fc3e63daf5cfe9bd4475257905903b78542
```

Comment:
why is this one not versioned?

Reply:
@drujensen because this 👉 https://github.com/amberframework/amber/pull/536 fix was after `v0.6.4`